README.md

```diff
@@ -81,31 +81,28 @@ element {
 
 ----
 
-## To add or edit icons and build the web font and demo page
+## Add / edit icons, build the web font and demo page
 
-Install the following:
+1. Install the following:
 
-* [fontforge](http://fontforge.org/)
-* [ttfautohint](http://www.freetype.org/ttfautohint/#download)
+	* [fontforge](http://fontforge.org/)
+	* [ttfautohint](http://www.freetype.org/ttfautohint/#download)
+	* [X11](http://support.apple.com/kb/ht5293) (Mac only)
 
-On a Mac <ahref="http://brew.sh/">Homebrew</a> is a popular package manager to install these programs. You'll also need <a href="http://support.apple.com/kb/ht5293">X11</a> on your Mac.
+2. Clone the repository and install dependencies:
 
+		git clone https://github.com/Financial-Times/o-ft-icons.git
+		cd o-ft-icons
+		npm install
 
-Clone this repo and at the command line, `cd` to the repo's directory and run:
+3. Add or edit an SVG file to the `svg` folder (see [SVG file naming rules](#svg-file-naming-rules)).
+4. Generate the web font from the SVG sources:
 
-	npm install
+		grunt
 
-Add or edit an SVG file to the `svg` folder (see SVG file naming rules).
+5. Generate the component's demos:
 
-Next, run:
-
-	grunt
-
-This will generate the web font from the SVG sources.
-
-Finally, run:
-
-	origami-build-tools demo --local
+		origami-build-tools demo --local
```

Comment:
change it to `obt demo` and mention if you want to see it locally, run `obt demo --runServer`?